src/app/app-client.ts

```diff
@@ -571,7 +592,7 @@ export class AppClient {
       pb.LocationAuthRequest,
       pb.LocationAuthResponse
     >(service.locationAuth.bind(service), req);
-    return response.toObject();
+    return response.toObject().auth;
```

Comment:
Following the previous functions, this return should be `return response.getAuth()?.toObject()`. The following functions after this one, however, follow this pattern of `response.toObject()._`. Wanted to point it out in case we had a preference.

Reply:
I had changed some of them to be `response.getFoo().toObject()` and kept some as `response.toObject().foo` because I had a slight inkling that converting the entire message to a JS object would be slower than getting only the attribute we want and then calling `toObject()` on that. However, so many of these responses only had a single attribute that I figured it probably didn't actually make much of a difference. 

I perhaps should've done some benchmarking instead of making assumptions...